report/verification/mes/tmap_1b.md

```diff
@@ -0,0 +1,135 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.2
+kernelspec:
+  display_name: vv-festim-report-env
+  language: python
+  name: python3
+---
+
+# TMAP7 V&V Val-1b
+
+```{tags} 1D, MES, transient
+```
+
+This verification case {cite}`ambrosek_verification_2008` from TMAP7's V&V document consists of a semi-infinite slab with no traps under a constant concentration $C_0$ boundary condition on the left.
+
++++
+
+## FESTIM Code
+
+```{code-cell}
+:tags: [hide-cell]
+
+import festim as F
+import numpy as np
+import sympy as sp
+from scipy.special import erf
+from matplotlib import pyplot as plt
+
+C_0 = 1 # atom m^-3
+D = 1 # 1 m^2 s^-1
```

Comment:
Just so you know in general, it is not advised to use ones in verification problem.

Imagine that, inside FESTIM, we don't have the $D$ coefficient (a bug) and the equation that is encoded is simply

$$
\partial c / \partial t = \nabla \cdot (\nabla c)
$$


This test would pass, but it shouldn't.

Reply:
Yes, I'd forgotten to be honest. I'll make sure to avoid that in our own cases.

Reply:
Yep 👍 Of course this follows the TMAP7 case